<launch>
  <rosparam command="load" file="$(find displayer_app)/cfg/RadarDisplayerRos.yaml" />
  <node name="displayer_app" pkg="displayer_app" type="main.py" />
  <node name="displayer_window" pkg="displayer_app" type="window.py" />
  <node name="hp_limit_helper" pkg="hp_limit_helper" type="hp_limit_helper_node" />
</launch>
